src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSource.cs

```diff
@@ -19,6 +19,8 @@ public class PackageSource : IEquatable<PackageSource>
         /// </summary>
         public const int DefaultProtocolVersion = 2;
 
+        public const bool DefaultAllowInsecureConnections = false;
```

Comment:
I don't see this as being very useful as a public API: https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#changes-to-apis

Reply:
Thanks! Fixed. It's no more a public API.

Reply:
Sorry, had to change it back into a public API, as [PackageSourceContextInfoFormatter.cs](https://github.com/NuGet/NuGet.Client/blob/dev-hengliu-optoutHttpWarning-addAllowInsecureConnectionsProperty/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/PackageSourceContextInfoFormatter.cs#L37) needs this.

Reply:
Thanks! Fixed.